src/main.cpp

```diff
@@ -51,6 +51,7 @@ void validateVideoResolution()
 
 int main(int argc, char *argv[])
 {
+	(void)argc; //UNUSED
```

Comment:
I'm not fond of this idiom. I would prefer the name simply be commented out in the parameter list.